### DISCUSSION TITLE
Adding issues for Project from repos needs a scrollbar

### DISCUSSION THREAD

**Discussion by judej** [community/community#8356] [Projects and Issues]
When I want to add issues to a Project, I type #reponame and the list of issues show up that I can add. But it is limited to 7or so issues and I cannot scroll down to the remaining issues I need to add from the repo.

**Comment by michaelwschultz** [comment]
Ahh! I thought some of my issues were just missing. Good catch! They are only populating 8 items it seems, wonder why this is. Makes adding issues really annoying because you have to actually search for each one.
